store.py

```diff
@@ -1,14 +1,14 @@
 ''' Implements a store of disutils PKG-INFO entries, keyed off name, version.
 '''
+
+from __future__ import print_function
+
 import sys, os, re, time, hashlib, random, types, math, stat, errno
 import logging, string, datetime, calendar, binascii, urllib, urllib2, cgi
 import posixpath
 from collections import defaultdict
 import cPickle as pickle
-try:
-    import psycopg2
-except ImportError:
-    pass
+import psycopg2
```

Comment:
I removed the except-pass anyway if fails later if one does not have psycopg2 working.